build.gradle

```
@@ -40,6 +40,16 @@ dependencies {
     implementation("com.github.Softawii:curupira:v1.0.0") {
         changing = true
     }
```

```suggestion
    implementation("com.github.Softawii:curupira:v1.0.0") {
        exclude module: 'slf4j-simple'
        changing = true
    }
    implementation 'org.apache.logging.log4j:log4j-slf4j2-impl:2.24.1'
```

Adicionar essas instruções para melhor a integração com o SLF4J do Curupira